evalml/tests/model_understanding_tests/test_permutation_importance.py

```diff
@@ -25,6 +25,7 @@ class DoubleColumns(Transformer):
     that in the future.
     """
     name = "DoubleColumns"
+    hyperparameter_ranges = {}
```

Comment:
Do you remember why this is needed?

Reply:
PipelineBase's `get_hyperparameters` has the line `component_hyperparameters = copy.copy(component_class.hyperparameter_ranges)` which fails if `DoubleColumns` does not have a `hyperparameter_ranges` attribute.

I think this was okay previously because we never tried to access the hyperparameters attribute--but now that it's done in init, we do and catch this!